crates/ty_python_semantic/resources/mdtest/generics/legacy/functions.md

```diff
@@ -455,3 +455,43 @@ def overloaded_outer(t: T | None = None) -> None:
     if t is not None:
         inner(t)
 ```
+
+## Unpacking a TypeVar
+
+We can infer precise heterogeneous types from the result of an unpacking operation applied to a
+TypeVar if the TypeVar's upper bound is a type with a precise tuple spec:
```

Comment:
very nit: I've only been using the camel-case spelling to refer to `TypeVar` the Python entity. In prose like this I've been using "type variable" or "typevar" (lowercase)